Kajabi/pine
# [DSS-390]: refactor() - Unify helperMessage and related prop naming

# Description

Updates some component's prop naming to match and adhere to agreed-upon conventions.

- Textarea: `hintMessage` > `helperMessage`
- Input: `hint` > `helperMessage`
- Input: `errorText` > `errorMessage`

There should be no visual changes to the components due to these updates.

Fixes [DSS-390](https://kajabi.atlassian.net/browse/DSS-390)

## Type of change

- [x] Refactor (Code changes that neither fix a bug nor add a feature)

# How Has This Been Tested?

1. Navigate to the components listed above and verify prop names are updated.
2. Verify documentation has been updated to match these updates.


**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.



[DSS-390]: https://kajabi.atlassian.net/browse/DSS-390?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
